contentcuration/contentcuration/tests/utils/test_cache.py

```diff
@@ -1,12 +1,12 @@
 import mock
-from django.test import SimpleTestCase
+from django.test import TestCase
 
 from ..helpers import mock_class_instance
 from contentcuration.models import ContentNode
 from contentcuration.utils.cache import ResourceSizeCache
 
 
-class ResourceSizeCacheTestCase(SimpleTestCase):
+class ResourceSizeCacheTestCase(TestCase):
```

Comment:
I did like the changes you initially showed me over changing this. The reason is that it keeps the scope of a test narrowed to only what's needed. Like these tests don't really need the database, as was evident by your mocking changes fixing the test failures. Also, database tests can add additional overhead and make the test suite take longer, so it can be an unnecessary performance hit if unneeded. Overall though, this is not a review blocker at all.

Reply:
I like the `TestCase` more because it makes the `node_fts` part be handled by Django natively. I am fine to change it to the precious one if you insist though :smile:

Reply:
Yeah I think changing the inherited class is an indirect resolution of the actual failure you ran into